tenets/codelingo/code-review-comments/go-error-fmt/codelingo.yaml

```diff
@@ -0,0 +1,40 @@
+funcs:
+  - name: isInvalidErrName
+    type: asserter
+    body: |
+      function (a) {
+        var dot = a.charAt(a.length - 1) === '.'
+        var cap = a.charAt(0) >= "A" && a.charAt(0) <= "Z"
+        return dot || cap
+      }
+tenets:
+  - name: go-error-fmt
+    flows:
+      codelingo/docs:
+        title: Go Error Format
+        body: |
+          Error strings should not be capitalized (unless beginning with proper nouns 
+          or acronyms) or end with punctuation, since they are usually printed following
+           other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"),
+            so that log.Printf("Reading %s: %v", filename, err) formats without a spurious 
```

Comment:
looks like some funky string formatting is going on here. i think this should all be the same indent

Reply:
"lingo run docs" looks fine. so you probably don't need to change this.